PNA.adoc

```diff
@@ -279,57 +287,61 @@ here as "main".
 | packet from   |               |                        |
 | host loopback |               |                        |
 |---------------|---------------|------------------------|
-~
+|===
 
 Note that each mirrored packet that results from `mirror_packet`
 operations will have its own next place that it will go to be
 processed, independent of the original packet, and independent of any
 other mirror copies made of the same original packet.
 
-# PNA Data types
+[#sec-pna-data-types]
+== PNA Data types
 
-## PNA type definitions {#sec-pna-type-definitions}
+[#sec-pna-type-definitions]
+=== PNA type definitions 
 
 Each PNA implementation will have specific bit widths in the data
 plane for the types shown in the code excerpt of Section
-[#sec-pna-type-definitions-code].  These widths are defined in the
+<<#sec-pna-type-definitions-code>>.  These widths are defined in the
 target specific `pna.p4` include file.  They are expected to differ
-from one PNA implementation to another[^PNAP4TargetSpecific].
+from one PNA implementation to another[1].
 
-[^PNAP4TargetSpecific]: It is expected that `pna.p4` include files for
+[1]: It is expected that `pna.p4` include files for
     different targets will be nearly identical to each
     other.  Besides the possibility of differing bit widths for these
     PNA types, the only expected differences between `pna.p4` files
     for different targets would be annotations on externs, etc. that
     the P4 compiler for that target needs to do its job.
 
-For each of these types, the P4 Runtime API[^P4RuntimeAPI] may use bit
+For each of these types, the P4 Runtime API[2] may use bit
```

Comment:
I think for a footnote whose content is just a link to some other document, like this one and a few others I will make comments on, it seems preferable to add a bibliography/references section, and use that markup here.